client/ayon_core/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,139 @@
+                tooltip=(
+                    "Whether to include all custom maya attributes found "
+                    "on nodes as attributes in the Alembic data."
+                )
+            ),
+        ]
+    )
+
+    return defs
+
+
+def convert_legacy_alembic_creator_attributes(node_data, class_name):
+    """This is a legacy transfer of creator attributes to publish attributes
+    for ExtractAlembic/ExtractAnimation plugin.
+    """
+    publish_attributes = node_data["publish_attributes"]
+
+    if class_name in publish_attributes:
+        return node_data
+
+    attributes = [
+        "attr",
+        "attrPrefix",
+        "visibleOnly",
+        "writeColorSets",
+        "writeFaceSets",
+        "writeNormals",
+        "renderableOnly",
+        "visibleOnly",
+        "worldSpace",
+        "renderableOnly"
+    ]
+    plugin_attributes = {}
+    for attr in attributes:
+        if attr not in node_data["creator_attributes"]:
+            continue
+        value = node_data["creator_attributes"].pop(attr)
+
+        plugin_attributes[attr] = value
+
+    publish_attributes[class_name] = plugin_attributes
+
+    return node_data
+
+
+class CreateAnimation(plugin.MayaHiddenCreator):
+    """Animation output for character rigs
+
+    We hide the animation creator from the UI since the creation of it is
+    automated upon loading a rig. There's an inventory action to recreate it
+    for loaded rigs if by chance someone deleted the animation instance.
+    """
+
+    identifier = "io.openpype.creators.maya.animation"
+    name = "animationDefault"
+    label = "Animation"
+    product_type = "animation"
+    icon = "male"
+
+    write_color_sets = False
+    write_face_sets = False
+    include_parent_hierarchy = False
+    include_user_defined_attributes = False
+
+    def read_instance_node(self, node):
+        node_data = super(CreateAnimation, self).read_instance_node(node)
+        node_data = convert_legacy_alembic_creator_attributes(
+            node_data, "ExtractAnimation"
+        )
+        return node_data
+
+    def get_instance_attr_defs(self):
+        super(CreateAnimation, self).get_instance_attr_defs()
+        defs = _get_animation_attr_defs(self)
+        return defs
+
+
+class CreatePointCache(plugin.MayaCreator):
+    """Alembic pointcache for animated data"""
+
+    identifier = "io.openpype.creators.maya.pointcache"
+    label = "Pointcache"
+    product_type = "pointcache"
+    icon = "gears"
+    write_color_sets = False
+    write_face_sets = False
+    include_user_defined_attributes = False
+
+    def read_instance_node(self, node):
+        node_data = super(CreatePointCache, self).read_instance_node(node)
+        node_data = convert_legacy_alembic_creator_attributes(
+            node_data, "ExtractAlembic"
+        )
+        return node_data
+
+    def get_instance_attr_defs(self):
+        super(CreatePointCache, self).get_instance_attr_defs()
```

Comment:
Whatever the `super` call returns here is unused - even if it did return any attribute definitions the returned result is unused.

Reply:
Fixed.